Soccar [Physics]/src/nl/soccar/physics/models/WheelPhysics.java

```diff
@@ -94,14 +95,14 @@ public WheelPhysics(float relPosX, float relPosY, float width, float height, boo
     }
 
     /**
-     * Apply force on the wheel based on the power of the car and the desired speed.
+     * Apply force on the wheel based on the power of the carPhysics and the desired speed.
      */
     public void updateDrive() {
 
         Vec2 currentForwardNormal = body.getWorldVector(new Vec2(0, 1));
 
         float currentSpeed = Vec2.dot(getForwardVelocity(), currentForwardNormal);
-        float force = PhysicsConstants.CAR_POWER * 10;
+        float force = power * 10;

```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Cast one of the operands of this multiplication operation to a "float". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS2184)